pom.xml

```diff
@@ -1528,6 +1549,31 @@
             <groupId>org.json</groupId>
             <artifactId>json</artifactId>
```

Comment:
the org.json artifacts shouldn't be in any ASF artifact, e.g [HADOOP-13794](https://issues.apache.org/jira/browse/HADOOP-13794). The hive team need to keep on top of that. And yes, exclusion was the correct action